WordPress/src/main/java/org/wordpress/android/workers/reminder/ReminderConfig.kt

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.workers.reminder
+
+import org.wordpress.android.workers.reminder.ReminderConfig.ReminderType.DAILY
+import org.wordpress.android.workers.reminder.ReminderConfig.ReminderType.WEEKLY
+import java.time.DayOfWeek
+import java.time.LocalDate
+import java.time.temporal.TemporalAdjusters.next
+
+sealed class ReminderConfig(val type: ReminderType) {
+    object DailyReminder : ReminderConfig(DAILY)
```

Comment:
Do we need the `DailyReminder`?

Reply:
You're right, we don't really need it right now. I just think having explicit models for each type of recurrence should make this much more flexible in the future, if we ever decide to build upon it with more advanced rules (interval, for example).